libs/books/data-access/src/constants/appconstants.ts

```diff
@@ -0,0 +1,4 @@
+export class AppConstants {
+  public static readingApi = '/api/reading-list';
+  public static bookSearchApi = '/api/books/search';
+}
```

Comment:
Created constants for better coding standards and concise code

Reply:
Suggestion: you can change the file name to `app-constants.ts`